### ISSUE TITLE
Feature Request - Global Rules to skip False Positives

### ISSUE THREAD

**Issue by vikas027** [ansible/ansible-lint#241]
# Issue Type
- Feature request

# Ansible and Ansible Lint details

```
# ansible --version
ansible 2.0.1.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides
#

# ansible-lint --version
ansible-lint 3.4.10
#
```

- ansible installation method: pip
- ansible-lint installation method: pip

# Desired Behaviour

It would be good to have a global ignore policy so that there is no need to use tag `skip_ansible_lint` everywhere in my code.

For example, in my organization, we tend to use the latest packages with `yum` module. This may be a problem for some but is absolutely fine with our pipeline. 

Another example is `command: '/sbin/reboot'` AFAIK, there isn't a module to reboot a Unix/Linux machine.
If we can have a something like `ansible-lint.ignore` file in which we can list out the things which we can ignore, then we won't have to edit the `yaml` files and place tag `skip_ansible_lint` everywhere.

**Comment by willthames** [comment]
I recommend https://github.com/willthames/ansible-review for more complicated workflows. That's not to say that I don't think ansible-lint.ignore wouldn't be nice, just that I don't have enough time to implement it myself (PRs welcome, following discussion on what that might look like)

**Comment by awcrosby** [comment]
Closing since configuration file allows global skipping of rules.
